Makefile

```diff
@@ -366,6 +368,16 @@ deploy-knative: create-cluster
 	kubectl wait  --for=condition=Ready=True KnativeServing/knative-serving -n knative-serving --timeout=$(TIMEOUT_SECS)
 	kubectl wait  --for=condition=Ready=True KnativeEventing/knative-eventing -n knative-eventing --timeout=$(TIMEOUT_SECS)
 	
+.PHONY: deploy-monitoring
+deploy-monitoring: create-cluster
```

Comment:
Instead of dropping grafana, we can move it to a separate target. It will not be used by e2e, but is useful for documenting the grafana related information. Let me know what you think.

Reply:
Documentation is here: https://github.com/apache/incubator-kie-kogito-docs/tree/main/serverlessworkflow/modules/ROOT/pages/cloud/operator